packages/mermaid/src/diagrams/railroad/railroadDiagram.ts

```diff
@@ -0,0 +1,15 @@
+import { DiagramDefinition } from '../../diagram-api/types.js';
+// @ts-ignore: jison doesn't export types
+import parser from './railroadParser.jison';
+import { db } from './railroadDB.js';
+import renderer from './railroadRenderer.js';
```

Comment:
```suggestion
import { renderer } from './railroadRenderer.js';
```

Use named import instead of default.

Reply:
Why?